src/main/java/de/javagl/geom/Shapes.java

```diff
@@ -57,7 +55,7 @@ static List<List<Point2D>> computeRegions(Shape shape, double flatness) {
         List<List<Point2D>> regions = new ArrayList<>();
         PathIterator pi = shape.getPathIterator(null, flatness);
         float[] coords = new float[6];
-        List<Point2D> region = Collections.emptyList();
```

Comment:
`emptyList()` was fine here because IIRC every segment always starts with `MOVETO`
(I'm currently reviewing changes in IDE so gonna fix this myself)